modules/indexing/src/main/java/org/apache/ignite/internal/visor/query/VisorQueryDetailMetrics.java

```diff
@@ -180,6 +205,8 @@ public long getLastStartTime() {
         out.writeDouble(avgTime);
         out.writeLong(totalTime);
         out.writeLong(lastStartTime);
+        U.writeString(out, schema);
+        out.writeObject(local);
```

Comment:
Why not out.writeBoolean?

Reply:
New fields have been removed. Failed to collect new metrics without compatibility break.